src/query/service/src/pipelines/processors/transforms/transform_udf_server.rs

```diff
@@ -164,13 +164,18 @@ impl TransformUdfServer {
         record_connect_external_duration(func.func_name.clone(), connect_duration);
 
         Profile::record_usize_profile(ProfileStatisticsName::ExternalServerRequestCount, 1);
+        record_running_requests_external_start(func.name.clone(), 1);
+        record_request_external_block_rows(func.func_name.clone(), num_rows);
```

Comment:
does this metric represents the rows of each request?

if it is, how about name it as `record_request_external_batch_rows`?